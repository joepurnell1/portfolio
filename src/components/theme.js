const colours = {
  white: '#FFFFFF',
  accentPink: '#E84855',
  lightPink: '#F7A0A7',
  fairlyDarkBlue: '#022e51',
  darkestBlue: '#012b4c',
  peach: '#FFA987',
  blue: '#067BC2',
  kindaGreen: '#00FFC5',

  // new
  pinkish: '#f77ca0',
  solidPinkish: '#ff5158',
  leblue: '#26ebe3',
};

export default {
  breakpoints: {
    small: '420px',
    medium: '720px',
    large: '1060px',
  },

  typography: {
    headerColour: colours.solidPinkish,
    heavyWeight: 600,
    textColour: colours.darkestBlue,
    smallTextColour: colours.darkestBlue,
    link: {
      standardColour: colours.darkestBlue,
      specialColour: colours.pinkish,
      specialHoverColour: colours.leblue,
      fancyLines: [colours.solidPinkish, '#35FF69', '#440381', '#008DD5', '#F038FF']
    }
  },

  project: {
    accentOneStart: 'rgba(247, 124, 160, 0.4)',
    accentOneEnd: 'rgba(247, 124, 160, 0.7)',
  },

  tag: {
    colours:  [colours.solidPinkish, '#2ee65d', '#440381', '#008DD5', '#F038FF'],
  },

  Global: {
    background: colours.white,
    textColour: colours.darkestBlue,
    strongTextWeight: 600,
  },
  Header: {
    accent: colours.accentPink,
  },
  Link: {
    accent: colours.accentPink,
  },
  Homepage: {
    highlightedBackground: colours.accentPink,
  },
  ColouredInfoBand: {
    backgroundColour: colours.accentPink,
    textColour: colours.white,
  },
  HeadedSection: {
    headerText: colours.accentPink,
  },
};