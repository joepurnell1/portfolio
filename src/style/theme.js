const colours = {
  white: '#FFFFFF',
  accentPink: '#E84855',
  lightPink: '#F7A0A7',
  darkestBlue: '#011627',
  yellow: '#F9DC5C',
};

export default {
  Link: {
    background: colours.white,
    accent: colours.accentPink,
  },
};
