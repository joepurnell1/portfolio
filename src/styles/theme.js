const colours = {
  white: '#FFFFFF',
  accentPink: '#E84855',
  lightPink: '#F7A0A7',
  fairlyDarkBlue: '#022e51',
  darkestBlue: '#011627',
  peach: '#FFA987',
  blue: '#067BC2',
  kindaGreen: '#00FFC5',
};

export default {
  typography: {
    headerColour: colours.accentPink,
    heavyWeight: 600,
    textColour: colours.darkestBlue,
    smallTextColour: colours.accentPink,
  },

  Global: {
    background: colours.white,
    textColour: colours.darkestBlue,
    strongTextWeight: 600,
  },
  Header: {
    accent: colours.accentPink,
  },
  Link: {
    accent: colours.accentPink,
  },
  Homepage: {
    highlightedBackground: colours.accentPink,
  },
  ColouredInfoBand: {
    backgroundColour: colours.accentPink,
    textColour: colours.white,
  },
  HeadedSection: {
    headerText: colours.accentPink,
  },
};
