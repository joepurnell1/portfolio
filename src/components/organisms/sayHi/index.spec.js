import React from 'react';
import renderer from 'react-test-renderer';
import { ThemeProvider } from 'styled-components';
import SayHi from './index';
import theme from '../../../styles/theme';

describe('say hi component', () => {
  it('should render default as expected', () => {
    const render = renderer.create(
      <ThemeProvider theme={theme}>
        <SayHi />
      </ThemeProvider>,
    );
    expect(render.toJSON()).toMatchSnapshot();
  });
});
