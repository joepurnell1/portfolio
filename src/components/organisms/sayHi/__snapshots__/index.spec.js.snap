// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`say hi component should render default as expected 1`] = `
<div
  className="sc-gZMcBi hHIYWE"
  name="sayHi"
>
  <div>
    <h1
      className="sc-htpNat dPgqoS"
    >
      Say Hi
    </h1>
    <h4
      className="sc-bwzfXH eNwIkd"
    >
      Want to collaborate, chat, or work together?
    </h4>
  </div>
  <div
    className="sc-jTzLTM jyRLJi"
  >
    <a
      className="sc-VigVT dqUZmV"
      href="mailto:joepurnell1@gmail.com"
    >
      <p
        className="sc-gqjmRU iduEac sc-bdVaJa dXPSEv"
      >
        joepurnell1@gmail.com
      </p>
    </a>
    <h4
      className="sc-fjdhpX ezvssc sc-bwzfXH eNwIkd"
    >
      or
    </h4>
  </div>
  <div
    className="sc-htoDjs hwdHvz"
  >
    <a
      className="sc-ifAKCX dAbayQ"
      href="https://github.com/joepurnell1"
      size="45px"
    >
      <img
        alt="github"
        className="sc-EHOje jaoqMH"
        height="40px"
        src="test-file-stub"
        width="40px"
      />
    </a>
    <a
      className="sc-ifAKCX dAbayQ"
      href="https://twitter.com/joepurnell8?lang=en"
      size="45px"
    >
      <img
        alt="twitter"
        className="sc-EHOje jaoqMH"
        height="40px"
        src="test-file-stub"
        width="40px"
      />
    </a>
    <a
      className="sc-ifAKCX dAbayQ"
      href="https://www.linkedin.com/in/joe-purnell-7294295b/"
      size="45px"
    >
      <img
        alt="linkedin"
        className="sc-EHOje jaoqMH"
        height="40px"
        src="test-file-stub"
        width="40px"
      />
    </a>
  </div>
</div>
`;
